eng/CodeChecks.ps1

```diff
@@ -1,82 +1,28 @@
-#requires -version 5
+#Requires -Version 6.0
```

Comment:
Since we use Join-Path with more than 2 parameters, that version of the cmdlet is only supported in 6.0 and up.